syntax = "proto3";
package TiebaDesktop;

import "proto/Error_RequestMgr.proto";
import "proto/PermissionList.proto";

message DataRes_GetUserBlackInfo {
    int32 is_black_white = 1;
    PermissionList perm_list = 2;
}

message GetUserBlackInfoResIdl {
    Error_RequestMgr error = 1;
    DataRes_GetUserBlackInfo data = 2;
}
